test/Makefile

```diff
@@ -1,6 +1,6 @@
 STDLIB ?= ../src
 MOC ?= moc
-WASMTIME_OPTIONS = --disable-cache --cranelift
+WASMTIME_OPTIONS = --disable-cache --enable-cranelift-debug-verifier
```

Comment:
So `--disable-cache` won't work without `--enable-cranelift-debug-verifier`? That sounds strange. What is the expected logic behind the combination of these two options?

Reply:
I believe `--disable-cache` should work regardless of `--enable-cranelift-debug-verifier`. I just added it to supplement `--cranelift` .  I don't know how `--cranelift` maps to the new version of wasmtime.

Reply:
Then let's remove the latter.
```suggestion
WASMTIME_OPTIONS = --disable-cache
```

Reply:
Please apply ^^

Reply:
ok updated